wasm/webapi/esm-integration/worker-import-source-phase.tentative.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
> We could alternatively make this a JS file test per the previous PR feedback for other platforms, but the HTML wrapper seems fine for now.

Using a JS file has the advantage that we do not need two separate tests (main thread and worker), but we can have a single `.any.js` that runs in both.

Reply:
Yes, the existing `.any.js` test will capture that, but I don't think we can test worker imports in this way since they require a worker created under `type: 'module'`, not a worker script which I think would be the assumption for `.any.js`?

Reply:
Our existing JS test already tests the source phase builtin invariants under workers though.